### DISCUSSION TITLE
Proposals on osu!direct background music

### DISCUSSION THREAD

**Discussion by LamGC** [ppy/osu#13037] [Ideas]
In osu!direct, when the music player is playing, I think the volume of music can be reduced, the pause can be kept unchanged during audition, and the original volume can be restored after closing osu!direct.
This can reduce the discomfort caused by switching between BGM and audition music.(For example, switching between fast and slow rhythms is not right.)

Original(*There may be some difficulties in translation, so the original version of issue is attached.*):
我认为在打开 osu!direct 时，当音乐播放器在播放时，bgm可以减小，在试听过程中暂停bgm的操作可以保持不变，不过需要在关闭 osu!direct 后恢复原来的音量。
这可以减少在背景音乐和试听音乐之间快速切换造成的违和感（例如，在快节奏和慢节奏音乐之间切换）。

**Comment by nyquillerium** [comment]
If I understood correctly, this is a proposal to keep the main track playing when previewing a track in osu!direct, albeit at a lower volume. The volume of the original track would fade to a near inaudible level, and the preview track would start playing.

**Comment by LamGC** [comment]
@nyquillerium This is it! :-D

**Comment by LamGC** [comment]
But everyone's volume may not be the same in practice, so if you can pause in the preview,it's better, that is, reduce the volume in the non-preview state, and pause in the preview. (emoji: Ponder)

**Comment by LastExceed** [comment]
People playing at different volume levels shouldn't be a problem, we can just use a percentage decrease instead of a fixed value

**Comment by peppy** [comment]
I agree with this and think it should be quite easy to implement. It feels wrong that the track stops completely while a preview is playing.
